Auth0/SafariWebAuth.swift

```diff
@@ -229,19 +234,8 @@ class SafariWebAuth: WebAuth {
                              nonce: self.nonce)
     }
 
-    var redirectURL: URL? {
-        let bundleIdentifier = Bundle.main.bundleIdentifier ?? SafariWebAuth.NoBundleIdentifier
-        var components = URLComponents(url: self.url, resolvingAgainstBaseURL: true)
-        components?.scheme = self.universalLink ? "https" : bundleIdentifier
-        return components?.url?
-            .appendingPathComponent("ios")
-            .appendingPathComponent(bundleIdentifier)
-            .appendingPathComponent("callback")
-    }
-
     func clearSession(federated: Bool, callback: @escaping (Bool) -> Void) {
         let logoutURL = federated ? URL(string: "/v2/logout?federated", relativeTo: self.url)! : URL(string: "/v2/logout", relativeTo: self.url)!
-        #if swift(>=3.2)
```

Comment:
I assume these version checks were here for a reason, and now the "else" cases for swift versions lower than 3.2 are gone on both login and logout. That looks like breaking change to me. What is the reason to remove them?

Reply:
We're no longer supporting Swift 3.x.